lib/ohai/plugins/linux/network.rb

```diff
@@ -384,6 +384,14 @@ def favored_default_route(routes, iface, default_route, family)
     counters Mash.new unless counters
     counters[:network] = Mash.new unless counters[:network]
 
+    # ohai.plugin[:network][:default_route_table] = 'default'
+    if configuration(:default_route_table).nil? || configuration(:default_route_table).empty?
+      default_route_table = 'main'
```

Comment:
Does populating attribute values from `route show` produce the same results as `route show table main`? If not, the default behavior of this plugin should probably be to shell out with `route show` by default.